docs/modules/ROOT/pages/deployment-modes.adoc

```diff
@@ -52,36 +65,194 @@ If you forget it, you have to remove the credentials file in _data/dbms/authoriz
 
 [[web-server]]
-== Neo4j Browser - Dedicated web server
-
+== Dedicated web server
 
 It is possible to host a dedicated web server to serve Neo4j Browser as a web application.
-See the github page for source code, https://github.com/neo4j/neo4j-browser.
-//(Is it available as an npm install package?)
+The Neo4j Browser is a client side single page application (SPA), you can serve the static content with a HTTP server.
+
+[NOTE]
+====
+If you have installed Neo4j on your system then disable the browser module (`BROWSER`) from running. +
+See link:https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/#config_dbms.http_enabled_modules[dbms.http_enabled_modules] configuration.
+====
+
+=== HTTP web servers
+
+Here is a list of suggestions for simple HTTP web servers that can be used to host the static content:
+
+Python3 `http.server`::
+Is a simple http server that is bundled with Python 3. Do not use in production environment. +
+https://docs.python.org/3/library/http.server.html
+[source, shell]
+----
+python -m http.server --directory neo4j-browser/ 8080
+----
+
+Node.js `http-server`::
+Is a simple, zero-configuration command-line http server. +
+https://www.npmjs.com/package/http-server
+[source, shell]
+----
+npx http-server -p 8080 neo4j-browser/
+----
+
+Node.js `serve`::
+Is a simple http server that is ideal for serving a static site, single page application or just a static file. +
+https://www.npmjs.com/package/serve
+[source, shell]
+----
+npx serve -l 8080 neo4j-browser/
+----
+
+[TIP]
+====
+HTTP servers suitable for production use:
+
+* Nginx -- is a web server that can also be used as a reverse proxy and a load balancer.
+* Apache HTTP Server.
+* Internet Information Services (IIS).
+====
+
+
+=== Get pre build and run web server
+
+You can download Neo4j Browser from Maven (link:https://search.maven.org/artifact/org.neo4j.client/neo4j-browser[`https://search.maven.org/artifact/org.neo4j.client/neo4j-browser`]) and serve the web application with a HTTP server.
+
+
+.Get pre build and run web server
+====
+
+**1. Download Neo4j Browser.**
+
+Download Neo4j Browser from Maven:
+
+[source, shell, subs="attributes+"]
+----
+wget https://repo1.maven.org/maven2/org/neo4j/client/neo4j-browser/{neo4j-browser-version}/neo4j-browser-{neo4j-browser-version}.jar
+----
+
+Or manually go to link:https://search.maven.org/artifact/org.neo4j.client/neo4j-browser[`https://search.maven.org/artifact/org.neo4j.client/neo4j-browser`] and download Neo4j Browser.
+
+**2. Unzip the .jar file.**
+
+Unzip the `.jar` file:
+
+[source, shell, subs="attributes+"]
+----
+unzip neo4j-browser-{neo4j-browser-version}.jar
+----
+
+**3. Host web application with a HTTP server.**
+
+Start a web server (in this example you will start a web server with Python 3) to serve the content:
+
+[source, shell]
+----
+python -m http.server --directory browser/ 8080
+----
+
+Point your web browser to `http://localhost:8080`.
```

Comment:
Again the link formatting. We have to decide how we format the links and make the documentation consistent.